File: libs/gradle/plugin-processor/tests/res/TransformPlugin.java
package com.satori.libs.gradle.plugin.processor;

import com.satori.libs.gradle.plugin.annotations.GradlePlugin;
import org.gradle.api.Plugin;
import org.gradle.api.Project;

@GradlePlugin("com.satori.transform")
public class TransformPlugin implements Plugin<Project> {
  
  @Override
  public void apply(Project project) {
  }
}

File: libs/gradle/transform/src/com/satori/libs/gradle/transform/TransformPlugin.kt
package com.satori.libs.gradle.transform

import com.satori.libs.gradle.plugin.annotations.*
import org.gradle.api.*

@GradlePlugin("com.satori.transform")
open class TransformPlugin : Plugin<Project> {
  
  override fun apply(project: Project) {
    val conv = TransformPluginConvention(project)
    project.convention.plugins["transform"] = conv
  }
}